dart/tool/dart/generate_dart.dart

```diff
@@ -169,12 +170,20 @@ abstract class Member {
   String toString() => name;
 }
 
-class Api extends Member {
+class Api extends Member with ApiParseUtil {
+  int serviceMajor;
+  int serviceMinor;
   List<Method> methods = [];
   List<Enum> enums = [];
   List<Type> types = [];
 
   void parse(List<Node> nodes) {
+    var serviceVersion = parseServiceVersion(nodes);
+    serviceMajor = serviceVersion[0];
+    serviceMinor = serviceVersion[1];
+    // TODO encode service version into generated dart
```

Comment:
+1. It would be great to have just the string available as a static field, to know what version the lib was generated for.